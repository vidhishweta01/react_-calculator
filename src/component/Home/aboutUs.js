import Navbar from '../navbar';

const Home = () => (
  <>
    <Navbar />
    <h1> Welcome to our page! </h1>
    <p>
      Lorem Ipsum is simply dummy text of the printing and typesetting industry.
      Lorem Ipsum has been the industrys standard dummy text ever since the
      1500s, when an unknown printer took a galley of type and scrambled it to
      make a type specimen book. It has survived not only five centuries
      {' '}
    </p>
    <p>
      Contrary to popular belief, Lorem Ipsum is not simply random text. It has
      roots in a piece of classical Latin literature from 45 BC, making it over
      2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney
      College in Virginia, looked up one of the more obscure Latin words,
      consectetur, from a Lorem Ipsum passage
    </p>
  </>
);

export default Home;
