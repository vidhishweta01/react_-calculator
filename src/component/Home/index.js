import Home from './aboutUs';

export default Home;
