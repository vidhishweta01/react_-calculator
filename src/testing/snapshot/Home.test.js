import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Home from '../../component/Home';

describe('Home component', () => {
  it('matches the snapshot', () => {
    const home = renderer.create(<BrowserRouter><Home /></BrowserRouter>).toJSON();
    expect(home).toMatchSnapshot();
  });
});
