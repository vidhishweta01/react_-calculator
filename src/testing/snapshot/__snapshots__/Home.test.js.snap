// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component matches the snapshot 1`] = `
Array [
  <nav
    className="nav"
  >
    <h1
      className="head"
    >
      Math Magicians
    </h1>
    <div
      className="links"
    >
      <a
        exact={true}
        href="/"
        onClick={[Function]}
      >
        Home
         
      </a>
      <a
        className="mid_link"
        href="/quote"
        onClick={[Function]}
      >
        Quote
         
      </a>
      <a
        href="/calculator"
        onClick={[Function]}
      >
        Calculator 
      </a>
    </div>
  </nav>,
  <h1>
     Welcome to our page! 
  </h1>,
  <p>
    Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries
     
  </p>,
  <p>
    Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical Latin literature from 45 BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia, looked up one of the more obscure Latin words, consectetur, from a Lorem Ipsum passage
  </p>,
]
`;
